apps/server/src/graphql/Task.ts

```diff
@@ -375,6 +388,58 @@ When the task is:
   })
 );
 
+builder.mutationField("addSubTask", (t) =>
+  t.prismaFieldWithInput({
+    type: "Task",
+    description: "Create a subtask.",
+    input: {
+      parentTaskId: t.input.id({ required: true, description: "The ID of the parent Task" }),
+      title: t.input.string({ required: true, description: "The title of the subtask." }),
+      status: t.input.field({
+        type: TaskStatusEnum,
+        defaultValue: "TODO",
+        description: "The status of the subtask.",
+      }),
+      durationInMinutes: t.input.field({
+        defaultValue: 60,
+        type: "PositiveInt",
+        description: "The length of time (in minutes) the task is expected to take.",
+      }),
```

Comment:
Oh sorry for the confusion, the snippet I gave in the ticket was an example usage and the values I gave were arbitrary. No need to add default values.
```suggestion
      durationInMinutes: t.input.field({
        type: "PositiveInt",
        description: "The length of time (in minutes) the task is expected to take.",
      }),
```